### DISCUSSION TITLE
Cannot start a project cloned from a repository in GitHub

### DISCUSSION THREAD

**Discussion by nunommcsilva** [community/community#21747] [New to GitHub]
<p>Hello,</p>
<p>I’ve cloned a project from a friend to my pc and I’ve tried to run it with Angular, but I can’t. It shows the following message:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/0919e79e2563e4d964c4b7e049892779677ca2ff.png" data-download-href="/uploads/short-url/1ivNtcB2O0IGfDnOjk8bu5qlqKr.png?dl=1" title="01.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/50527320/181066524-641d39aa-388a-4f00-b827-ca57b41447aa.png" alt="01.PNG" data-base62-sha1="1ivNtcB2O0IGfDnOjk8bu5qlqKr" width="690" height="159" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/0919e79e2563e4d964c4b7e049892779677ca2ff_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">01.PNG</span><span class="informations">1158×267 7.05 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Do you know how to solve this problem?</p>
<p>Thanks in advance!</p>

**Comment by gautamkrishnar** [comment, marked as answer]
<p>Type the following commands:</p>
<pre><code>npm install
npm install -g @angular/cli
</code></pre>
<p>It should work. This not anything related to github. it is related to the project’s dependancies.</p>
